tests/unit/Entity/ItemIdSetTest.php

```diff
@@ -16,6 +16,35 @@
  */
 class ItemIdSetTest extends \PHPUnit_Framework_TestCase {
 
+	public function testGetIterator() {
+		$set = new ItemIdSet();
+		$iterator = $set->getIterator();
+
+		$this->assertInstanceOf( 'Traversable', $iterator );
+	}
+
+	/**
+	 * @dataProvider serializationsProvider
+	 */
+	public function testGetSerializations( array $expected ) {
```

Comment:
Two main reasons:

* It makes the tests more complex, harder to change, and harder to maintain
* You are more likely to repeat the same mistake (ie missing a case you should handle) in both places